pages/docs/tutorials/getting-started/request-reply.md

```diff
@@ -0,0 +1,238 @@
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example with replier
+  version: 1.0.0
+  description: Simple example with a replier that replies to the request.
+
+channels:
+  // Same as for the requester
+
+operations:
+  pongReply:
+    action: receive
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      channel: 
+        $ref: '#/channels/pong'
+```
+
+# Sub-patterns in request/reply
+
+In the simple example above we saw how you could set up a request/reply pattern across two applications, where the request/reply happened over the same channel `/` on an unknown server and protocol, which could have been HTTP, Kafka, or WebSocket, in this simple example it does not matter, cause the only difference would be how the server information is defined.
+
+However, there are sub-patterns to request/reply that AsyncAPI v3 supports, let's take a look at them. 
+
+## Request/reply over different channels
+If you come from a REST or WebSocket environment, this sub-pattern might seem unfamiliar, but in the event-driven world of Kafka or NATS this is a common pattern to utilize where you do the request over one channel, and reply on a different one.
+
+In this example, the reply is on a statically defined channel, so you, at design time, know exactly where the reply is returned to.
+
+The only difference in this AsyncAPI document, in relation to the simple example is that each channel has now been given a different address `/ping` and `/pong` respectively.
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example for a requester with static reply channel
+  version: 1.0.0
+  description: Example with a requester that initiates the request/reply pattern on a different channel than the reply is using.
+
+channels:
+  ping:
+    address: /ping
+    messages:
+      ping:
+        $ref: '#/components/messages/ping'
+  pong:
+    address: /pong
+    messages:
+      pong:
+        $ref: '#/components/messages/pong'
+
+operations:
+  pingRequest:
+    action: send
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      channel: 
+        $ref: '#/channels/pong'
+```
+
+Defining the **replier** is the same as for the requester, again using the `receive` action instead as the only difference.
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example for replier with static reply channel
+  version: 1.0.0
+  description: Example with a replier that returns the response on a different channel than the request happened on.
+
+channels:
+  // Same as for the requester
+
+operations:
+  pongReply:
+    action: receive
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      channel: 
+        $ref: '#/channels/pong'
+```
+
+## Request/reply with dynamic response channel
+
+A second sub-pattern is where we do not know the reply channel at design time, but instead, it's dynamic and determined at runtime. This could for example be using the request message payload or header to dictate the response channel.
```

Comment:
```suggestion
A second sub-pattern is where we do not know the reply channel at design time, but instead, it's dynamic and determined at runtime. This could, for example, be using the request message payload or header to dictate the response channel.
```